great-idea-website/css/index.css

```diff
@@ -63,4 +63,140 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+header .logo {
+
+	display: inline-block;
+	margin-left: 20px;
+
+}
+
+header nav a{
+	text-decoration: none ;
+	color:gray;
+
+}
+
+
+
+
+header nav {
```

Comment:
for the nav I would try a width of 100% and also do a display: block which will take up the whole top layer of the layer of the layout. Also with a position: fixed to keep it up their as you are scrolling.